api/src/main/resources/messages.properties

```diff
@@ -182,7 +182,7 @@ coreapps.noActiveVisit.description=This patient is not in an active visit. If th
 
 coreapps.deadPatient = The patient is deceased - {0} - {1}
 coreapps.deadPatient.description=This patient is deceased. If you wish to view or edit details of a previous visit, choose a visit from the list on the left.
-coreapps.markPatientDead.label=Mark Patient Dead
+coreapps.markPatientDead.label=Mark Patient Deceased
```

Comment:
I don't know where this is used, but although "dead" is probably clearer I think "deceased" is less "intense" and more professional if that makes any sense.